### PR TITLE
[SETUPAPI] cfgmgr.c: Use WideCharToMultiByte(WideCharCount=-1)

### PR DESCRIPTION
**Pull request by SergeGautherie**
when actual string length is unknown, not the full buffer length.

## Purpose

Consistency.

Follow-up to #5943.

## Proposed changes

- Use WideCharToMultiByte(WideCharCount=-1)

### CODE REVIEW

**Comment by SergeGautherie** [comment]
"Usual"
`fatal error C1051: program database file, 'D:\a\reactos\reactos\build\modules\rostests\apitests\user32\CMakeFiles\user32_apitest.dir\vc140.pdb', has an obsolete format, delete it and recompile`

**Review by HBelusca — APPROVED** [review]

File: dll/win32/setupapi/cfgmgr.c [hunk 1]
```diff
@@ -4352,7 +4352,7 @@ CM_Get_Device_Interface_List_ExA(
     if (WideCharToMultiByte(CP_ACP,
                             0,
                             BufferW,
-                            BufferLen,
+                            -1,
```
Comment (TAN-Gaming) [inline dll/win32/setupapi/cfgmgr.c:4355]: The same as `CM_Get_Device_ID_List_ExA`.

**Review by TAN-Gaming — CHANGES_REQUESTED** [review]

File: dll/win32/setupapi/cfgmgr.c [hunk 2]
```diff
@@ -3232,7 +3232,7 @@ CM_Get_DevNode_Custom_Property_ExA(
             *pulLength = WideCharToMultiByte(CP_ACP,
                                              0,
                                              BufferW,
-                                             lstrlenW(BufferW) + 1,
+                                             ulLengthW,
```
Comment (TAN-Gaming) [inline dll/win32/setupapi/cfgmgr.c:3235]:
```suggestion
                                             ulLengthW / sizeof(WCHAR),
```

File: dll/win32/setupapi/cfgmgr.c [hunk 3]
```diff
@@ -3442,7 +3442,7 @@ CM_Get_DevNode_Registry_Property_ExA(
             *pulLength = WideCharToMultiByte(CP_ACP,
                                              0,
                                              BufferW,
-                                             lstrlenW(BufferW) + 1,
+                                             LengthW,
```
Comment (TAN-Gaming) [inline dll/win32/setupapi/cfgmgr.c:3445]:
```suggestion
                                             LengthW / sizeof(WCHAR),
```

**Pull request closed without merging**